wpilibj/src/athena/java/edu/wpi/first/wpilibj/internal/HardwareTimer.java

```diff
@@ -65,7 +65,7 @@ public double getMatchTime() {
      * Create a new timer object. Create a new timer object and reset the time to zero. The timer is
      * initially not running and must be started.
      */
-    public TimerImpl() {
+    TimerImpl() {
```

Comment:
https://stackoverflow.com/questions/243218/should-we-declare-a-public-constructor-when-the-class-is-declared-as-package-pri